### PR TITLE
[#318] Add conflict with "doctrine/dbal:2.7.0"

### LINKED ISSUE #318: All fields on the revision table are marked as not null.
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| Used Version     | 1.0.8

The revision table that is created during the schema generation is marking all of the fields as "NOT NULL", even though a lot of them are nullable. This is causing integrity constraint violations when a revision is saved.

### PR DESCRIPTION
**Pull request by phansys**
|Q            |A     |
|---          |---   |
|Branch       |master|
|Bug fix?     |yes   |
|New feature? |no    |
|BC breaks?   |no    |
|Deprecations?|no    |
|Tests pass?  |yes   |
|Fixed tickets|#318  |
|License      |MIT   |
|Doc PR       |n/a   |

**TODO:**
- [x] Add tests.

Fixes #318.

### CODE REVIEW

**Comment by phansys** [comment]
Changed base to `1.0` since it should be treated as a bugfix IMO.

File: composer.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
         "fabpot/php-cs-fixer": "^1.11"
     },
     "conflict": {
+        "doctrine/dbal": "2.7.0",
```
Comment (axzx) [inline composer.json]: `"doctrine/dbal": ">2.7.0",`

Comment (phansys) [reply composer.json]: Release [2.7.1](https://github.com/doctrine/dbal/compare/v2.7.0...v2.7.1#diff-688c20898c96ed1017c40a3f9b12d051R135) should fix the issue. Could you confirm you're experiencing it with that version?

Comment (axzx) [reply composer.json]: OK, I'm sorry. 
I thought that from 2.7.0 is this problem.

**Comment by tolry** [comment]
great, thanks!

**Comment by phansys** [comment]
@tolry, thank you for merging!
Release [1.0.9](https://github.com/simplethings/EntityAuditBundle/releases/tag/v1.0.9) states that this PR was merged in it, but it isn't.

**Comment by tolry** [comment]
hmm, lgtm https://github.com/simplethings/EntityAuditBundle/blob/v1.0.9/composer.json#L26 or am I missing something?

**Comment by tolry** [comment]
@phansys d'oh - you used short array syntax in your test  :smile: https://travis-ci.org/simplethings/EntityAuditBundle/jobs/364493226

**Pull request merged**